requirements_all.txt

```diff
@@ -543,3 +543,7 @@ yahooweather==0.8
 
 # homeassistant.components.zeroconf
 zeroconf==0.17.6
+
+# homeassistant.components.sensehat
+# homeassistant.components.sensor.sensehat
+sense-hat==2.2.0
```

Comment:
You need to run `script/gen_requirements_all`, as specified in the PR instructions